File: src/leetcode/python/RangeAdditionII_598.py
class RangeAdditionII_598(object):
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        mn, mx = m, n
        for r in ops:
            mn = min(mn, r[0])
            mx = min(mx, r[1])
        return mn * mx

File: src/leetcode/java/RangeAdditionII_598.java
package leetcode.java;

public class RangeAdditionII_598 {
  public int maxCount(int m, int n, int[][] ops) {
    int min = m;
    int max = n;
    for (int[] row : ops) {
      min = Integer.min(min, row[0]);
      max = Integer.min(max, row[1]);
    }
    return min * max;
  }
}
